src/common-utils/index.js

```diff
@@ -38,23 +38,24 @@ function constructAdditionalParams(options){
             }
         }
         else {
-            params += `itemFilter(${count}).name=${key}&`;
+            filterParams += `itemFilter(${count}).name=${key}&`;
             if (!Array.isArray(value)) {
-                params += `itemFilter(${count}).value=${value}&`;
+                filterParams += `itemFilter(${count}).value=${value}&`;
             } else {
                 for (let innerKey in value) {
-                    params += `itemFilter(${count}).value(${innerKey})=${value[innerKey]}&`;
+                    filterParams += `itemFilter(${count}).value(${innerKey})=${value[innerKey]}&`;
                 }
             }
             if(key === "MinPrice" || key === "MaxPrice"){
-                params += `itemFilter(${count}).paramName=Currency&
+                filterParams += `itemFilter(${count}).paramName=Currency&
                 itemFilter(${count}).paramValue=${currency[currencyKey]}&`;
```

Comment:
@EarthlingDavey  Thanks for this PR. is it possible to write test case for this ?.

Reply:
Oh man, thanks for catching the rookie mistake. I'll write a test

Reply:
@pajaydev I've updated the expected results from the existing tests. I don't think I need to write a new one. Please advise. Thanks.

Reply:
@EarthlingDavey  Thanks, I appreciate it